Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
@JoshEngebretson That would definitely work for the editor.  For the player though, individuals would still need to add the ```require("AtomicEventLoop")``` to their project's ```main.js``` themselves.  One possibility though is to update the new project template to automatically have this in the ```main.js``` that gets generated.  What do you all think of going down that route?